kubejs/server_scripts/packmode.js

```diff
@@ -1,18 +1,16 @@
 //priority: 1004
-const setMode = (player) => {
-    console.log('setting mode for player: ' + player);
+PlayerEvents.loggedIn((event) => {
+    const expertModeQuest = '2CCCDD05AED3153F';
+    const normalModeQuest = '0E45018D600614BA';
+
+    console.log(`Setting mode for player: ${event.player}`);
+
     if (global.packmode == 'expert') {
-        // player.data.ftbquests.complete('0000000000000FEB');
+        event.player.data.ftbquests.reset(normalModeQuest);
+        event.player.data.ftbquests.complete(expertModeQuest);
     } else {
-        // player.data.ftbquests.reset('0000000000000FEB');
-    }
-};
-
-ServerEvents.highPriorityData((event) => {
-    if (event.server) {
```

Comment:
@NielsPilgaard `event.server` here is perpetually undefined, not sure what the best approach is in this case, but I went with a different event entirely. I suspect this may have some un-intended consequences, however, that I'm not seeing yet.

Reply:
I think it's `event.getServer()`

Reply:
I think I tried that and it said it wasn't valid. Will try again though.

Reply:
Yeah, it doesn't like that at all 

`! [22:53:06] [ERR] #23: Error occurred while handling event 'ServerEvents.highPriorityData': TypeError: Cannot find function getServer in object dev.latvian.mods.kubejs.script.data.DataPackEventJS@528e75ef. (server_scripts:packmode.js#23)`

It's ok. I think this'll be OK for now and I can come back to it if necessary.